chapter_recurrent-modern/index_vn.md

```diff
@@ -30,7 +34,8 @@ Finally, we will take machine translation as an example, and introduce the encod
 recurrent neural networks and modern practices for such sequence to sequence learning problems.
 -->
 
-*dịch đoạn phía trên*
+Trên thực tế, phần lớn các bài toán học chuỗi như nhận dạng giọng nói tự động, chuyển đổi văn bản thành giọng nói và dịch máy đều có đầu vào và đầu ra là các chuỗi với chiều dài bất kì.
+Cuối cùng, ta sẽ lấy ví dụ về dịch máy, giới thiệu kiến trúc mã hóa - giải mã (*encoder-decoder*) dựa trên mạng nơ-ron truy hồi và các phương pháp hiện đại để giải quyết bài toán học từ chuỗi sang chuỗi.
```

Comment:
`sequence to sequence` cũng nên có một chỗ trong glossary nhỉ? cc: @lkhphuc @minhduc0711

Reply:
#1699 nhé.